src/panels/config/zha/zha-groups-dashboard.ts

```diff
@@ -0,0 +1,79 @@
+import "../../../layouts/hass-subpage";
+import "./zha-groups-data-table";
+
+import {
+  LitElement,
+  html,
+  TemplateResult,
+  property,
+  customElement,
+  CSSResult,
+  css,
+} from "lit-element";
+import { HomeAssistant } from "../../../types";
+import { haStyleDialog } from "../../../resources/styles";
+import { ZHAGroup, fetchGroups } from "../../../data/zha";
+import { sortZHAGroups } from "./functions";
+
+@customElement("zha-groups-dashboard")
+export class ZHAGroupsDashboard extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public narrow = false;
+  @property() public _groups!: ZHAGroup[];
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this._fetchGroups();
+  }
+
+  protected render(): TemplateResult {
+    return html`
+      <hass-subpage
+        header=${this.hass.localize(
+          "ui.panel.config.zha.groups.zha_zigbee_groups"
+        )}
+      >
+        <div class="content">
+          <zha-groups-data-table
+            .hass=${this.hass}
+            .narrow=${this.narrow}
+            .groups=${this._groups}
+            class="table"
+          ></zha-groups-data-table>
+        </div>
+      </hass-subpage>
+    `;
+  }
+
+  private async _fetchGroups() {
+    this._groups = (await fetchGroups(this.hass!)).sort(sortZHAGroups);
+  }
+
+  static get styles(): CSSResult[] {
+    return [
+      haStyleDialog,
+      css`
+        .content {
+          padding: 4px;
+        }
+        zha-groups-data-table {
+          width: 100%;
+        }
+        .button {
+          float: right;
+        }
+
+        .table {
+          height: 200px;
+          overflow: auto;
```

Comment:
Why not combine these with `zha-groups-data-table`?

Reply:
I did it like this in case I want to use it in another view with different sizes (later in the series of PR's for this there is a devices one that I use several times). I can move it if preferred.